resolver/src/dns_lru.rs

```diff
@@ -63,7 +63,7 @@ impl DnsLru {
         let ttl_until = now + ttl;
 
         // insert into the LRU
-        let lookup = Lookup::new(Arc::new(rdatas));
+        let lookup = Lookup::new_with_ttl(Arc::new(rdatas), ttl);
```

Comment:
per @briansmith recommendation, it seems ttl_until is what is wanted here, no? I missed that before